### PR TITLE
Fix NXP-25114 adding missing param

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,5 @@ npm-debug.log
 .DS_Store
 .screenshots/
 
+*.iml
+*.idea
```
Comment (Gabez0r) [inline .gitignore]: This is IDE specific, so we should rather have these exceptions locally, on a global .gitignore file.

Comment (Gabez0r) [reply .gitignore]: On the other hand, we have it on other projects, so perhaps we can keep it, just rename `*.idea` to `.idea`, since it's a folder.

**Comment by Gabez0r** [comment]
👍

**Comment by guirenard** [comment]
:+1:

**Pull request merged**